zebrad/tests/acceptance.rs

```diff
@@ -12,15 +12,28 @@ use toml;
 use zebra_test::prelude::*;
 use zebrad::config::ZebradConfig;
 
-pub fn tempdir(create_config: bool) -> Result<(PathBuf, impl Drop)> {
+#[derive(PartialEq)]
+enum ConfigMode {
+    NoConfig,
+    Ephemeral,
+    Persistent,
+}
+
+fn tempdir(config_mode: ConfigMode) -> Result<(PathBuf, impl Drop)> {
     let dir = TempDir::new("zebrad_tests")?;
 
-    if create_config {
+    if config_mode != ConfigMode::NoConfig {
         let cache_dir = dir.path().join("state");
         fs::create_dir(&cache_dir)?;
 
         let mut config = ZebradConfig::default();
+        // We need to set cache_dir regardless of ephemeral because we run tests
+        // in parallel on non-linux platforms and database will be saved in the
+        // default Zebra cache dir, and that will cause cause conflicts
         config.state.cache_dir = cache_dir;
```

Comment:
If we avoid setting path when ephemeral is true, we won't need to set `cache_dir`:
https://github.com/ZcashFoundation/zebra/pull/939/files#r475633200